docs/changelog-fragments.d/776.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed test_http_over_https_error on Solaris -- by :user:`mtelka`.
```

Comment:
Let's improve highlighting and we'll be good to go.
https://cheroot--776.org.readthedocs.build/en/776/history/#to-be-included-in-the-next-release
```suggestion
Fixed ``test_http_over_https_error`` on Solaris -- by :user:`mtelka`.
```